instant-distance-py/src/lib.rs

```diff
@@ -9,14 +9,15 @@ use std::iter::FromIterator;
 use instant_distance::Point;
 use pyo3::conversion::IntoPy;
 use pyo3::exceptions::{PyTypeError, PyValueError};
-use pyo3::proc_macro::{pyclass, pymethods, pymodule};
 use pyo3::types::{PyList, PyModule, PyString};
+use pyo3::{pyclass, pymethods, pymodule};
 use pyo3::{Py, PyAny, PyErr, PyObject, PyRef, PyRefMut, PyResult, Python};
 use serde::{Deserialize, Serialize};
 use serde_big_array::BigArray;
 
 #[pymodule]
-fn instant_distance(_: Python, m: &PyModule) -> PyResult<()> {
+#[pyo3(name = "instant_distance")]
+fn instant_distance_py(_: Python, m: &PyModule) -> PyResult<()> {
```

Comment:
Why not keep the function name `instant_distance` here?

Reply:
PyO3 now creates a module with the same name as this function, which caused conflicts with the Rust crate. This seems cleaner than prefixing with :: everywhere.